geometry_msgs/Point position
geometry_msgs/Quaternion orientation
std_msgs/ColorRGBA color